### ISSUE TITLE
Jobs!

### ISSUE THREAD

**Issue by mpmdean** [shenmidelin/edrixs#7]
Maksim: Initialize new repo under the NSLS-II organize with the cookiecutter template. Add all of us.
Stuart: Look at scipy and how this compiles Fortran in terms of generating a good package structure. 
Ylin: Put edrixs into a newly organized repo
Stuart: How to get core hole life and the element edge energy e.g. hephaestus? Or has this been superseded. Make a json file? Talk to Eli or Bruce?
Stuart: Look at how python calls fortran https://docs.scipy.org/doc/numpy/f2py/

**Comment by mrakitin** [comment]
The skeleton was created at https://github.com/NSLS-II/edrixs.

I left a comment in one of the multi-language (C core with Python, Ruby, and Fortran interfaces) projects: https://github.com/atztogo/spglib/issues/54#issuecomment-470320188. We may use that project's structure as an example.

**Comment by shenmidelin** [comment]
Thanks @mrakitin .  spglib is a very good example, we can learn its structure. I am creating a new branch (devel) of edrixs on my account, and reshape its structure similar to spglib.

**Comment by mpmdean** [comment]
Perhaps we can directly work in https://github.com/NSLS-II/edrixs rather than making another branch your @shenmidelin 's account?

**Comment by shenmidelin** [comment]
I'm OK with that, so I just clone https://github.com/NSLS-II/edrixs to my local computer, make changes 
and git push ?

**Comment by mrakitin** [comment]
@shenmidelin, yes, I agree with @mpmdean that working in the new "skeleton" repo https://github.com/NSLS-II/edrixs makes sense. Working in a separate branch sounds OK. You can also work in your fork and then propose the changes via a PR. That way you can setup CI under your account and test/package independently before submitting to the main repo.

**Comment by mpmdean** [comment]
Hi @shenmidelin .

I think you should fork https://github.com/NSLS-II/edrixs  to your github account. Clone that to your computer, push back up to github and then pull that into https://github.com/NSLS-II/edrixs.

Is that reasonably clear? I can help you with the workflow if needed.

**Comment by shenmidelin** [comment]
Hi @mpmdean, thanks !
I understand that workflow. I will work like that.

**Comment by mrakitin** [comment]
And to avoid the names collisions, the existing repo under your account - https://github.com/shenmidelin/edrixs - should probably be renamed first, otherwise there will be an attempt to create the second repo under your account with the same name.

**Comment by shenmidelin** [comment]
OK, thanks !
